tests/nfs/nfs_operations.py

```diff
@@ -88,6 +104,16 @@ def cleanup_cluster(clients, nfs_mount, nfs_name, nfs_export):
     if not isinstance(clients, list):
         clients = [clients]
 
+    # Check nfs coredump
+    if ceph_cluster_obj:
+        nfs_nodes = ceph_cluster_obj.get_nodes("nfs")
+        coredump_path = "/var/lib/systemd/coredump"
+        for nfs_node in nfs_nodes:
+            if not check_coredump_generated(nfs_node, coredump_path, setup_start_time):
```

Comment:
It should be """ if check_coredump_generated """ right @MohitBis ?
Beacuse the method `check_coredump_generated` returns False or True based on whether the coredump is generated. Here, as per this logic, the execption is raised when coredump is not generated

Reply:
Thanks @pranavprakash20 
Updated